src/os_win/os_fs.c

```diff
@@ -361,8 +361,8 @@ __win_handle_read(
 		if (!ReadFile(fh->filehandle, addr, chunk, &nr, &overlapped))
 			WT_RET_MSG(session,
 			    nr == 0 ? WT_ERROR : __wt_getlasterror(),
-			    "%s: handle-read: ReadFile: failed to read %"
-			    WT_SIZET_FMT " bytes at offset %" PRIuMAX,
+			    "%s: handle-read: ReadFile: failed to read %lu "
```

Comment:
@agorrod, I believe a `DWORD` is typedef'd as an `unsigned long` on Windows, and both are 4B.

https://msdn.microsoft.com/en-us/library/aa383751(VS.85).aspx